runtime/onert/backend/cpu/KernelGenerator.cc

```diff
@@ -182,11 +182,11 @@ void KernelGenerator::visit(const ir::OpSequence &op_seq)
 {
   assert(!_return_fn_seq);
   assert(_tensor_builder->dynamicTensorManager());
-  assert(_tensor_builder->tensorRegistry());
+  assert(_tensor_reg);
```

Comment:
It seems to be a `TensorRegistry` of prev `OpSequence`. If it is the `TensorRegistry` of the other backend, is it okay?